backend/frontend/templates/frontend/user/board/view_partial_action_card_moved.html

```diff
@@ -1,5 +0,0 @@
-{% include "frontend/user/board/partial_card.html" %}
-
-<script>
```

Comment:
This should work for moving a card from any column to any other column. It would be best to just use this script as is. It does work so long as the card status is correct (and the card status should be correct)